cognite_toolkit/cdf_tk/load.py

```diff
@@ -855,12 +863,46 @@ def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_R
                         if item.external_id == ext_id:
                             items.remove(item)
                 try:
-                    return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+                    extractionPipelineList = ExtractionPipelineList(self.client.extraction_pipelines.create(items))
                 except Exception as e:
                     print(f"[bold red]ERROR:[/] Failed to create extraction pipelines.\n{e}")
                     self.ToolGlobals.failed = True
                     return None
-            return None
+
+        file_name = filepath.stem.split(".", 2)[1]
+        config_file_stem = f"{file_name}.config"
+        config_file = next(
+            (file for file in Path(filepath.parent).iterdir() if file.is_file() and config_file_stem in file.name),
+            None,
+        )
+
+        if not config_file.exists():
+            print(
+                f"  [bold yellow]WARNING:[/] no config file for extraction pipeline found. Expected to find {config_file_stem} in same folder as {file_name}"
+            )
+            return extractionPipelineList
+
+        resources = load_yaml_inject_variables(config_file, {})
```

Comment:
Should we assume extraction pipeline configs are yaml? You do not do that above, when you do not assume any file ending?